ac/ac-induction/src/mergeFun.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import { shuffle } from 'lodash';
+
+export default (obj: Object, dataFn: Object) => {
+  const listIni = [];
+  for (let i = 0; i < obj.config.examples.length; i += 1) listIni.push(i);
```

Comment:
Don't use a for loops if you can avoid it. In general, you never need to create an empty list and fill it up within a for loop. Especially for the simple case of `[1, 2, 3, 4, 5, ...]` 
```
const listIni = [...obj.config.examples.keys()]
const listIni = [...Array(obj.config.examples.length).keys()]
```